### PR TITLE
EDSC-2686: Fixes bug with prepopulated values

### PR DESCRIPTION
**Pull request by macrouch**

### CODE REVIEW

File: cypress/integration/unit/components/FormElement_spec.js [hunk 1]
```diff
@@ -50,12 +50,16 @@ function readXml(file) {
...
   const ui = uiResult.iterateNext()
-  const modelResult = document.evaluate('//*[local-name()="instance"]/*', doc)
+  const modelResult = document.evaluate('//*[local-name()="instance"]', doc)
   const model = modelResult.iterateNext()
 
   const resolver = buildXPathResolverFn(doc)
 
-  return { model, ui, resolver }
+  return {
+    model: model.firstElementChild,
```
Comment (abbottry) [inline cypress/integration/unit/components/FormElement_spec.js:60]: Places like this really deserve comments (`firstElementChild ignores comment blocks`)

File: cypress/integration/unit/util/buildParentXpath_spec.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+describe('buildParentXpath', () => {
+  it('concatenates parent and child xpath', () => {
+    const parent = 'parent'
+    const child = 'child'
+
+    expect(buildParentXpath(parent, child)).to.eq('parent/child')
+  })
+
+  it('does not concatenate parent and child xpath if child is absolute', () => {
```
Comment (abbottry) [inline cypress/integration/unit/util/buildParentXpath_spec.js]: Might seem minor, but assuming this catches `//` as well, can we add a test for that?

File: src/index.js [hunk 3]
```diff
@@ -151,12 +149,14 @@ export const EDSCEchoform = ({
     }
   }
 
+  if (!model.firstElementChild) return null
```
Comment (abbottry) [inline src/index.js:153]: Comment on what this is looking for would be nice. If there is no firstElementChild what does that mean?

File: src/util/buildParentXpath.js [hunk 4]
```diff
@@ -0,0 +1,14 @@
+/**
+ * Concatenates a parent xPath and child xPath, unless the child xPath is absolute
+ * @param {*} parentXpath
```
Comment (abbottry) [inline src/util/buildParentXpath.js]: Update these comments to actual types for params

**Review by trevorlang — APPROVED** [review]
I agree with Ryans comments. Other than that, things look good to me.

**Review by abbottry — APPROVED** [review]

**Pull request merged**